lib/greet.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+var greet = exports = module.exports = function(name) {
+  console.log('hello ' + name);
```

Comment:
I would have put the `console.log` into my index.js file, but it works here, too